### ISSUE TITLE
Integrate with Simple Tab Groups

### ISSUE THREAD

**Issue by gagarine** [rNeomy/auto-tab-discard#58]
[Simple tab groups](https://github.com/Drive4ik/simple-tab-groups) let you group tab and only show tab from the active group.

You can quickly end up with a lot of tab, and auto tab discard can help a lot. Simple tab groups can discard tab when they are hidden, yet it's not practical if you switch quickly between two tab groups. You want some kind of delay.

So some integration would be great. In particular, I would like to discard tab that was hidden for a number of seconds. This approach should work with all addon that hide tab using the API https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/tabs/hide

In the settings we will have an extra settings "discards hidden tab after XXX seconds" 

see also https://github.com/Drive4ik/simple-tab-groups/issues/201

**Comment by gagarine** [comment]
@rNeomy yes. Personally I will not discard "normal tabs" except if I have more than ~50. But I do want to discard "hidden tab" from groups that I don't use for weeks sometimes even if I have 5 of them in this given group.

Perhaps there are other use cases or ways to integrate that. Something like "discards all tabs from group that was not active for XXX second". But in fact it would have the same as treating hidden tabs differently. I'm right?

**Comment by rNeomy** [comment]
I added a new option to discard a tab if its state changes to hidden. Unfortunately, there is no way to remove the discarding from the extension side. So I guess it is not as useful as it could be with the reverse function.

**Comment by grahamperrin** [comment]
For context, a visual overview of some of what's currently available with Simple Tab Groups: 

![image](https://user-images.githubusercontent.com/192271/101439507-89401080-390c-11eb-8d74-44d910477d45.png)

![image](https://user-images.githubusercontent.com/192271/101439696-fbb0f080-390c-11eb-8c72-2765028efd57.png)

**Comment by grahamperrin** [comment]
> … want to discard "hidden tab" from groups that I don't use for weeks sometimes …

Correct me, please, if I'm wrong: if a (Simple Tab Groups) group is not in use when Firefox starts, then its (hidden) tabs will be in a discarded state from the outset.

**Comment by Chealer** [comment]
@gagarine : Please retitle as an issue report and clarify that this only affects Firefox. For example:

> Firefox: No integration with Simple Tab Groups extension
